test/core/elem.wast

```diff
@@ -24,15 +36,41 @@
   (elem (table $t) (i32.const 0) func $f $f)
   (elem (table $t) (offset (i32.const 0)) func)
   (elem (table $t) (offset (i32.const 0)) func $f $f)
+  (elem (offset (i32.const 0)))
+  (elem (offset (i32.const 0)) funcref (ref.func $f) (ref.null))
+  (elem (offset (i32.const 0)) func $f $f)
+  (elem (offset (i32.const 0)) $f $f)
+  (elem (i32.const 0))
+  (elem (i32.const 0) funcref (ref.func $f) (ref.null))
+  (elem (i32.const 0) func $f $f)
+  (elem (i32.const 0) $f $f)
 
-  (elem (i32.const 3) funcref (ref.func $f) (ref.func $g) (ref.func $f))
-  (elem (i32.const 3) funcref (ref.func $f) (ref.null) (ref.func $f))
-
-  (elem funcref (ref.func $f) (ref.func $g) (ref.func $f))
-  (elem funcref (ref.func $f) (ref.func $g) (ref.null))
-
-  (elem (table $t) (i32.const 3) funcref (ref.func $f) (ref.func $g) (ref.func $f))
-  (elem (table $t) (i32.const 3) funcref (ref.func $f) (ref.null) (ref.func $g))
+  (elem $a1 (table $t) (i32.const 0) funcref)
+  (elem $a2 (table $t) (i32.const 0) funcref (ref.func $f) (ref.null))
+  (elem $a3 (table $t) (i32.const 0) func)
+  (elem $a4 (table $t) (i32.const 0) func $f $g)
+  (elem $a5 (table $t) (i32.const 0) funcref)
+  (elem $a6 (table $t) (i32.const 0) funcref (ref.func $f) (ref.null))
+  (elem $a7 (table $t) (i32.const 0) func)
+  (elem $a8 (table $t) (i32.const 0) func $f $g)
+  (elem $a9 (table $t) (offset (i32.const 0)) funcref)
+  (elem $a10 (table $t) (offset (i32.const 0)) func $f $g)
+  (elem $a11 (table 0) (i32.const 0) func)
+  (elem $a12 (table 0x0) (i32.const 0) func $f $f)
+  (elem $a13 (table 0x000) (offset (i32.const 0)) func)
+  (elem $a14 (table 0) (offset (i32.const 0)) func $f $f)
+  (elem $a15 (table $t) (i32.const 0) func)
+  (elem $a16 (table $t) (i32.const 0) func $f $f)
+  (elem $a17 (table $t) (offset (i32.const 0)) func)
+  (elem $a18 (table $t) (offset (i32.const 0)) func $f $f)
+  (elem $a19 (offset (i32.const 0)))
+  (elem $a20 (offset (i32.const 0)) funcref (ref.func $f) (ref.null))
+  (elem $a21 (offset (i32.const 0)) func $f $f)
+  (elem $a22 (offset (i32.const 0)) $f $f)
+  (elem $a23 (i32.const 0))
+  (elem $a24 (i32.const 0) funcref (ref.func $f) (ref.null))
+  (elem $a25 (i32.const 0) func $f $f)
+  (elem $a26 (i32.const 0) $f $f)
```

Comment:
Also here you add tests for active element segments with IDs which are the same as the tests above. What is the purpose of these tests? As far as I understand, you cannot even use active elements by ID. Can you add a comment which explains the idea behind these tests, and how they are supposed to increase coverage?

Reply:
Like above, this is purely a syntax test, see heading.

Active elements can be used by ids, but behave like a dropped segment, i.e., trap in most cases.